### PR TITLE
Allow tables and sections to collapse

### PR DESCRIPTION
**Pull request by zbeucler-largo**
Allows both tables and sections to collapse individually

- [x] Sections
- [x] data tables
- [x] prop-defined tables

This is not a high priority merge request

### CODE REVIEW

File: base_app/src/app/app-tab-normal/app-tab-normal.component.css [hunk 1]
```diff
@@ -356,7 +357,7 @@ button, input[type=button], html input[type=button], input[type=reset], input[ty
     border-right-color: #B8B8B8;
     border-top-color: #FFFFFF;
     border-bottom-width: 1px;
-    -webkit-appearance: none;
+    /*-webkit-appearance: none;*/
```
Comment (tompatman) [inline base_app/src/app/app-tab-normal/app-tab-normal.component.css:360]: If not used, remove.

File: base_app/src/app/app-tab-normal/app-tab-normal.component.css [hunk 2]
```diff
@@ -375,7 +376,7 @@ button, input[type=button], html input[type=button], input[type=reset], input[ty
...
```
Comment (tompatman) [inline base_app/src/app/app-tab-normal/app-tab-normal.component.css:379]: Remove.

File: base_app/src/app/app.component.ts [hunk 3]
```diff
@@ -615,9 +617,10 @@ export class AppComponent implements OnInit, AfterViewInit /*, OnChanges */ {
         try {
             if ((cycle * this._refreshRate) > this._milliSecondsBeforeAutoPageReload) {
                 sessionStorage.autoReload = 'true';
-                setTimeout(() => {
-                    document.location.reload();
-                }, 1);
+                //setTimeout(() => {
```
Comment (tompatman) [inline base_app/src/app/app.component.ts:620]: Won't this break the automatic reload functionality. I do want to get it out of the system eventually, but I don't think we can do that until we've proven all memory leaks are resolved.

File: base_app/src/app/dataset-tables/dataset-table.ts [hunk 4]
```diff
@@ -83,24 +83,26 @@ export class DatasetTableComponent implements OnDestroy {
...
+        if (this.app._globalProps && this.app._globalProps._hiddenTables) {
             const tableId = this.getTableId();
-            const pos = this.app.globalProps._hiddenTables.indexOf(tableId);
+            const pos = this.app._globalProps._hiddenTables.indexOf(tableId);
             if (pos === -1) {
-                this.app.globalProps._hiddenTables.push(tableId);
+                this.app._globalProps._hiddenTables.push(tableId);
                 this._hidden = true;
-                console.log('toggle(' + tableId + '): hidden');
+                //console.log('toggle(' + tableId + '): hidden');
```
Comment (tompatman) [inline base_app/src/app/dataset-tables/dataset-table.ts:98]: Is there a file logger available with log levels?

**Review by tompatman — CHANGES_REQUESTED** [review]

File: base_app/src/app/app.component.ts (hunk 3 above)
Comment (zbeucler-largo) [reply base_app/src/app/app.component.ts]: Good catch, this was left over from some testing and should definitely not be commented out

**Pull request merged**